system-tests/project-fixtures/next/cypress/support/component.ts

```diff
@@ -1 +1,4 @@
-//
+// Importing global styles fails with Next.js due to restrictions on style imports.
```

Comment:
Do we want to add some kind of assertion that the global style is actually applied to some element?

Reply:
Added an assertion here: b0e8bf010a648f4ceb44a9c2d77c4d5aec09caba